### PR TITLE
Continue the tutorial, chapters 21-28

### PR DESCRIPTION
**Pull request by nkrkv**
TIL today: while reviewing docs on GitHub it is handy to click the “View” button above a file diff to see how the new page looks like with pictures, screencasts, etc.

### CODE REVIEW

File: docs/tutorial/23-count/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<div class="ui segment note">
+<span class="ui ribbon label">Note</span>
+This is a web-version of a tutorial chapter embedded right into the XOD IDE.
+To get a better learning experience we recommend to install the
+<a href="/downloads/">desktop IDE</a> or start the
+<a href="/ide/">browser-based IDE</a>, and you’ll see the same tutorial there.
+</div>
+
+Let’s make a funky wall clock. We now have all we need to do this.
```
Comment (brusherru) [inline docs/tutorial/23-count/README.md]: "We now have all..." -> "Now we have all..."?

File: docs/tutorial/23-count/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The clock will be the servo. Using the `count` node, we will change the degree
```
Comment (brusherru) [inline docs/tutorial/23-count/README.md]: "clock" -> "watch arrow"?

Comment (evgenykochetkov) [reply docs/tutorial/23-count/README.md]: "watch hand"

File: docs/tutorial/23-count/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+of its rotation from 0° to 180° in one minute.
```
Comment (evgenykochetkov) [inline docs/tutorial/23-count/README.md]: "the degree of its rotation" -> just "its rotation"?

File: docs/tutorial/21-pulses/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+The behavior of the `UPD` pin can also be set in the Inspector. For a pot the
+choice means:
+
+* `Never`: Never produce pulses, i.e. Do not take readings from the analog port
+  at all.
+* `On boot`: Generate a pulse once at startup. We will update the state exactly
+  once.
+* `Continuously`: Generate pulses constantly, i.e. take the readings from the
+  analog port with the highest possible frequency.
```
[the remark this answers is not in the document]
Comment (nkrkv) [reply docs/tutorial/21-pulses/README.md]: Your suggestions are OK, but why don’t you like “frequency”? Do you think it is too formal?

[the remark this answers is not in the document]
Comment (nkrkv) [reply docs/tutorial/21-pulses/README.md]: Nevermind. “Rate” is a more correct term in this context. Fixed it.

**Pull request merged**